README.rst

```diff
@@ -12,9 +12,27 @@ on a given codebase.
 Getting Started - Development
 -----------------------------
 
+Running using Docker
+~~~~~~~~~~~~~~~~~~~~
+
+Set SECRET_KEY environment variable in ``docker-compose.yml`` to some random string.
```

Comment:
Any example shell command that would generate this random string suitable for SECRET_KEY ?